src/main/resources/application.properties

```diff
@@ -1 +1,24 @@
+server.port= 8085
+## Stripe API
+stripe.key.publishable=pk_test_ZPLk0o0siEhURnWP3O31PAxl003mdYeZGh
+stripe.key.secret=sk_test_68AFqEufTbTSfjsBBmPC8hdd00xKfiq7ze
 
+## PostgreSQL
+spring.datasource.platform=postgresql
+spring.datasource.url=jdbc:postgresql://localhost:5432/payment
+spring.datasource.username=root
+spring.datasource.password=666
+
+
+#drop n create table again, good for testing, comment this in production
+
+#spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.generate-ddl=true
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database=postgresql
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL10Dialect
+spring.liquibase.change-log=classpath:/database/changelog-master.xml
+spring.liquibase.drop-first=true
```

Comment:
You just dropped all the benefits of Liquibase 🙂Liquibase was created for migrations e.g. safely modify already existing tables. With this setting set up, Liquibase will always drop the tables completely and recreate them.